addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4746,6 +4746,15 @@ export class OdooEditor extends EventTarget {
     _onTableMenuTogglerClick(ev) {
         const uiWrapper = closestElement(ev.target, '.o_table_ui');
         uiWrapper.classList.toggle('o_open');
+
```

Comment:
I'm a bit confused with the commit message saying the row menu UI appeared on the right side in RTL, while the previous commit states it appeared on the left. I'm guessing you're talking about two different elements here. Is there a way to make that clearer?

Reply:
It's because the previous commit message is about the table UI, not the row menu UI.
![](https://i.imgur.com/u23pZnv.png)

The table UI should be on the right of the table while the menu UI should be on the left of the table UI.

Reply:
Maybe call one the "table UI toggler" in the commit message so it's clearer?

Reply:
done